recipes/jenkins.rb

```diff
@@ -225,6 +225,7 @@ def jenkins = Jenkins.getInstance()
     not_if 'test -d /root/acme.sh'
   end
   execute 'acmesh-install' do
+    environment 'HOME' => '/root'
```

Comment:
Could be an alternative solution to use the `user` property and set it to `root`?

Reply:
On Linux, the user/group field will change the effective user/group ID but does not set up the environment as if that user is logged in and running the command.

I tend not to specify `user "root"` in recipes that would not work if they are run directly as a non-root user.